diff --git a/modules/network/src/tray_network_plugin.py b/modules/network/src/tray_network_plugin.py
index d36206c..2254ae1 100644
--- a/modules/network/src/tray_network_plugin.py
+++ b/modules/network/src/tray_network_plugin.py
@@ -194,11 +194,12 @@ class TrayNetworkPlugin(object):
         else:
             self.gui.remove_net("mobile")
 
+        def change_to_loading():
+            self.change_status_icon('loading')
+            self.let_rotate(True)
+
         if nm_module.nm_remote_settings.get_vpn_connections():
             self.gui.show_net("vpn")
-            def change_to_loading():
-                self.change_status_icon('loading')
-                self.let_rotate(True)
             self.gui.vpn_list.connecting_cb = change_to_loading
             self.active_vpn = None
             self.no_vpn_connecting = False
